packages/venia-ui/lib/RootComponents/Product/product.js

```diff
@@ -7,6 +7,7 @@ import { fullPageLoadingIndicator } from '../../components/LoadingIndicator';
 import ProductFullDetail from '../../components/ProductFullDetail';
 import getUrlKey from '../../util/getUrlKey';
 import mapProduct from '../../util/mapProduct';
+import ErrorView from '@magento/venia-ui/lib/components/ErrorView';
```

Comment:
Minor thing, I am not sure if we have this documented somewhere or if this is just an accepted thing in our code base but package imports should go above relative imports. I might be wrong, use your best guess.

Reply:
Yeah, that's a convention we have. I think we've said we don't want to reject PRs on it unless there's tooling for it though.

It's a "try your best to keep it consistent" kind of thing. If we _really_ care about it we'll write and enforce some lint rules.

Reply:
I'll update all these imports to use the absolute and then order based on distance from the file, as we've discussed prior.

Reply:
Updated.